### ISSUE TITLE
Parity check between V1 and V2, list all V2 issues

### ISSUE THREAD

**Issue by pkudinov** [near/queryapi#278]
- [ ] Feed indexers in not working in V2

**Comment by morgsmccauley** [comment]
Data for all indexers has now been migrated from V1 database to V2. The structure of the database (in relation to how Hasura communicates with it) has slightly changed, so I couldn't simply point the V2 infrastructure at the V1 database. Instead, I had to manually copy the contents of each database across.

For reference, here are the steps I followed:
1. **Stopped `coordiantor`** - so that indexer messages are no longer written to either V1 or V2
2. Waited for the SQS queue to drain - after which, the V1 database is essentially 'frozen'
3. Stopped `runner` - as it had massive backlogs of messages to process, I wanted to prevent it communicating with the V2 database
4.

**Comment by morgsmccauley** [comment]
Misclick

**Comment by morgsmccauley** [comment]
Data for all indexers has now been migrated from V1 database to V2.

The structure of the database (in relation to how Hasura communicates with it) has slightly changed, so I couldn't simply point the V2 infrastructure at the V1 database. Instead, I had to manually copy the contents of each database across.

For reference, here are the steps I followed:
1. **Stopped `coordiantor`** - so that indexer messages are no longer written to either V1 or V2
2. **Waited for the SQS queue to drain** - after which, the V1 database is essentially 'frozen'
3. **Stopped `runner`** - as it had massive backlogs of messages to process, I wanted to prevent it communicating with the V2 database
4. **For each indexer database within Postgres:**
    1. `pg_dump` the contents of the V1 database
    2. Apply the dump to the V2 database
5. **Delete all Redis streams** - After applying the dumps above, V2 has essentially caught up to where `coordinator` had stopped, therefore we can delete the existing messages and expect the `coordinator` will push only new and relevant messages
6. Verify Hasura metadata - to ensure that it can communicate to the Database correctly

I've done some sanity checks, i.e. checking counts of various tables, inspecting ordered rows to ensure they match, but I will do a more thorough check tomorrow.
